### PR TITLE
vm-repair : Adding feature for nested vm using hyperv

### PR DESCRIPTION
**Pull request by haagha**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
vm-repair

File: src/vm-repair/azext_vm_repair/custom.py [hunk 1]
```diff
@@ -164,6 +167,32 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
                                   .format(g=repair_group_name, disk_name=copy_disk_name, vm_name=repair_vm_name, uri=copy_disk_id)
             _call_az_command(attach_disk_command)
 
+        # invoke enable-NestedHyperV.ps1 again to attach Disk to Nested
+        if enable_nested:
+            logger.info("Running hyperv")
+
+            stdout, stderr = _invoke_run_command("win-enable-nested-hyperv.ps1", repair_vm_name, repair_group_name, 0)
```
Comment (qwordy) [inline src/vm-repair/azext_vm_repair/custom.py:174]: Does it support Linux environment?

Comment (haagha) [reply src/vm-repair/azext_vm_repair/custom.py:174]: No, this is only supported on windows. I will add the check in validation.

Comment (haagha) [reply src/vm-repair/azext_vm_repair/custom.py:174]: Added validation.

File: src/vm-repair/azext_vm_repair/_validators.py [hunk 2]
```diff
@@ -70,6 +70,10 @@ def validate_create(cmd, namespace):
     else:
         logger.debug('The source VM\'s OS disk is not encrypted')
 
+    if namespace.enable_nested:
+        if is_linux:
+            raise CLIError('Nested Vm is not supported for linux vm')
```
Comment (qwordy) [inline src/vm-repair/azext_vm_repair/_validators.py:75]:
```suggestion
            raise CLIError('Nested VM is not supported for Linux VM')
```

File: src/vm-repair/azext_vm_repair/custom.py [hunk 3]
```diff
@@ -164,6 +167,32 @@ def create(cmd, vm_name, resource_group_name, repair_password=None, repair_usern
...
+            logger.debug("stderr: %s", stderr)
+            if stderr:
+                logger.debug(stderr)
+                raise ScriptReturnsError('error when running script')
```
Comment (qwordy) [inline src/vm-repair/azext_vm_repair/custom.py:178]:
```suggestion
                raise ScriptReturnsError('Error when running script')
```

**Review by qwordy — APPROVED** [review]

**Pull request merged**